src/components/schema-ui/schema-menu.js

```diff
@@ -30,10 +38,11 @@ const SchemaMenu = ({ title, forPath, load, back, next }) => {
           <li key={`relationship-${index}`}>
             <button
               className="link--next"
-              onClick={() => loadNext(relationship.name)}
+              onClick={handleClick(name)}
```

Comment:
But, I think I see an error, I think it should it be this
```suggestion
              onClick={handleClick(relationship.name)}
```